src/renderer/component/speechUpload/view.jsx

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import lbry from "lbry";
+import FormField from "component/formField";
+import { Form, FormRow, Submit } from "component/form.js";
+import Link from "component/link";
+import Modal from "modal/modal";
+import * as modals from "constants/modal_types";
+import * as status from "constants/upload";
+
+class SpeechUpload extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      thumbnailUploadPath: "",
+    };
+  }
+
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.uploadStatus === status.ERROR) {
+      this.props.alertError("Upload failed. Please try again.");
+      this.props.resetUpload();
+      this.refs.thumbnail.val = null;
+    }
+  }
+
+  render() {
+    const { openModal } = this.props;
+
+    return (
+      <div>
+        {this.props.uploadStatus === status.MANUAL ? null : (
```

Comment:
can't you combine this and the below into one turnary operation?

i.e.
`this.props.uploadStatus === status.MANUAL ? <text input goes here> : <file select input goes here>`